inc/Composer/Plugin.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace HM\Platform\CMS\Composer;
+
+use Composer\Composer;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+use Composer\EventDispatcher\EventSubscriberInterface;
+
+class Plugin implements PluginInterface, EventSubscriberInterface {
+	public function activate( Composer $composer, IOInterface $io ) {
+	}
+	public static function getSubscribedEvents() : array {
+		return [
+			'post-update-cmd' => [ 'on_post_update_cmd' ],
+			'pre-install-cmd' => [ 'on_post_update_cmd' ],
+		];
+	}
+
+	public function on_post_update_cmd() {
```

Comment:
Since this is used for multiple events, something more descriptive would be better.